### PR TITLE
[BP][FLINK-33964][pulsar][docs] Remove dead link

### PR DESCRIPTION
**Pull request by GOODBOY008**
Remove dead link

### CODE REVIEW

**Comment by GOODBOY008** [comment]
@tisonkun  PTAL, Backport to v4.1 branch.

**Review by tisonkun — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -50,3 +50,13 @@ jobs:
       flink_version: ${{ matrix.flink }}
       timeout_global: 120
       timeout_test: 80
+  CheckDeadLinks:
```
Comment (tisonkun) [inline .github/workflows/ci.yml:53]: Please revert this change and do it in another patch. We may review #78 first. Pleaase avoid squash commits in cherry-pick.

**Review by tisonkun — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (GOODBOY008) [reply .github/workflows/ci.yml]: @tisonkun I just check this branch with no dead link. I had drop this commit.

**Review by tisonkun — APPROVED** [review]
